en/docs/reference/samples/api-development-sample/development-of-developer-optimized-apis-sample.md

```diff
@@ -70,21 +67,22 @@ For example if the API developer needs to restrict sending the address element o
 ```
 
 The API developer can add the following custom sequences to the API definition to achieve the latter mentioned scenario.
-In the following custom IN-sequence, it reads the User-Agent header and stores it in a custom property in WSO2 API Manager.
 
 **In-sequence**
 
+In the following custom IN-sequence, it reads the User-Agent header and stores it in a custom property in WSO2 API Manager.
+
 ``` java
     <sequence xmlns="http://ws.apache.org/ns/synapse" name="custom-in">
```

Comment:
The first indentation of the code should start inline with ```
Please fix this in other places as well

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/1762/commits/25c0d6d698df4049f8d4a78b186ae4377dfe9977